pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -90,13 +90,12 @@ type InstallationSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
-	Type                 string         `json:"type"`
-	RoutingSubdomain     string         `json:"routingSubdomain"`
-	MasterURL            string         `json:"masterUrl,omitempty"`
-	NamespacePrefix      string         `json:"namespacePrefix,omitempty"`
-	SelfSignedCerts      bool           `json:"selfSignedCerts"`
-	PullSecret           PullSecretSpec `json:"pullSecret"`
-	UseExternalResources bool           `json:"useExternalResources"`
```

Comment:
@dimitraz have we ensured that, when running e2e tests against a local cluster, a configmap is created that ensures all work will be done in-cluster instead of on AWS?

Reply:
No, forgot about that! Just added it in now. 
I was also thinking for the integreatly team for reviews it might make their lives easier to set the config defaults to openshift for now, while we do want to default to openshift? 
https://github.com/integr8ly/cloud-resource-operator/pull/56

Reply:
No, forgot about that! Just added it in now.
I was also thinking for the integreatly team for reviews it might make their lives easier to set the config defaults to openshift for now, while we do want to default to openshift?
integr8ly/cloud-resource-operator#56

Reply:
@dimitraz I'd agree with that, going to approve, can we create a ticket in the backlog to revert it once we have testing figured out for AWS stuff